design/one-pager-host-aware-stack-manager.md

```diff
@@ -0,0 +1,211 @@
+│   ├── stacks.crossplane.io_clusterstackinstalls.yaml
+│   ├── stacks.crossplane.io_stackinstalls.yaml
+│   ├── stacks.crossplane.io_stacks.yaml
+│   ├── storage.crossplane.io_buckets.yaml
+│   ├── workload.crossplane.io_kubernetesapplicationresources.yaml
+│   └── workload.crossplane.io_kubernetesapplications.yaml
+├── templates
+│   ├── NOTES.txt
+│   ├── _helpers.tpl
+│   ├── crossplane-admin-clusterrole.yaml
+│   ├── crossplane-clusterrole.yaml
+│   ├── crossplane-clusterrolebinding.yaml
+│   ├── crossplane-serviceaccount.yaml
+│   ├── environment-personas-clusterroles.yaml
+│   ├── namespace-personas-clusterroles.yaml
+│   ├── stack-manager-clusterrolebinding.yaml
+│   └── stack-manager-serviceaccount.yaml
+├── values.yaml
+└── values.yaml.tmpl
+```
+
+`crossplane-controllers`: this is the piece that will be deployed into Host Cluster. This chart includes deployments for
+ `crossplane` and `stack-manager` deployments and also RBACs necessary for `stack-manager` on Host Cluster.
+
+```
+.
+├── Chart.yaml
+├── templates
+│   ├── NOTES.txt
+│   ├── _helpers.tpl
+│   ├── crossplane-deployment.yaml
+│   ├── stack-manager-deployment.yaml
+│   ├── stack-manager-host-role.yaml
+│   ├── stack-manager-host-rolebinding.yaml
+│   └── stack-manager-host-serviceaccount.yaml
+├── values.yaml
+└── values.yaml.tmpl
+```
+
+Once we drop helm 2 support, we can create an umbrella chart for Crossplane which has dependency to these two charts and
+use the same definitions for regular installations as well. One problem that we need to solve in this case would be to
+define `requirements.yaml` to point correct helm repo/channel. This is left out of scope of this document.
+
+### Configuring a Pod against Tenant Instance API Server
+
+A pod created inside a Kubernetes Cluster can contact Api Server as follows:
+
+- Connection information made available with `KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT` environment
+  variables.
+- Authentication information is automatically put inside the pod at a predefined path according to assigned
+  `service account`.
+
+See [accessing the API from a pod](https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod)
+for more details.
+
+To configure a Pod against Tenant Instance API Server:
+
+1. Set `KUBERNETES_SERVICE_HOST` environment variable in pod definition as the name of the service backed by Tenant
+   Kubernetes API Server.
+2. Set `KUBERNETES_SERVICE_PORT` environment variable as the port of Tenant Instance API Server, e.g. `6443`
+3. To [opt out of automounting API credentials](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server)
+    1. Set `spec.automountServiceAccountToken` as false
+    2. Unset `spec.serviceAccountName`
+    3. Unset `spec.deprecatedServiceAccount`
+4. Mount service account token secret copied from CC to path
```

Comment:
what does `CC` mean here? i don't see this acronym introduced earlier in the doc

Reply:
It was meant to be Crossplane Cluster, which is `Tenant Instance API Server` in this context. Updated accordingly.